### PR TITLE
[MOV]: l10n_sa l10n_sa_invoices: merge l10n_sa_invoices into l10n_sa

### PR DESCRIPTION
**Pull request by JulienAlardot**
### [MOV]: l10n_sa l10n_sa_invoices: merge l10n_sa_invoices into l10n_sa
The l10n_sa_invoices changes do not need to be in a separate module as
they are always required for l10n_sa.
This merges the two modules into l10n_sa

**Task:** task-2723443
Signed-off-by: Julien Alardot (jual) <jual@odoo.com>

**Description of the issue/feature this PR addresses:** There are two modules for Saudi Arabia invoices that are always used together and never alone

**Current behavior before PR:** l10n_sa_invoices takes care of Saudi arabia invoices rendering requirements and l10n_sa take care of the rest

**Desired behavior after PR is merged:** Exactly the same as before, only using one module for that

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_sa/__manifest__.py [hunk 1]
```diff
@@ -2,18 +2,14 @@
...
-Odoo Arabic localization for most arabic countries and Saudi Arabia.
-
-This initially includes chart of accounts of USA translated to Arabic.
-
-In future this module will include some payroll rules for ME .
+Odoo Arabic localization for most Saudi Arabia.
 """,
     'website': 'http://www.dvit.me',
-    'depends': ['account', 'l10n_multilang'],
+    'depends': ['l10n_multilang', 'l10n_gcc_invoice'],
```
Comment (william-andre) [inline addons/l10n_sa/__manifest__.py:12]:
```suggestion
    'depends': [
        'l10n_multilang',
        'l10n_gcc_invoice',
    ],
```
Same for the PoS module

File: addons/l10n_sa/__manifest__.py [hunk 2]
```diff
@@ -2,18 +2,14 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Saudi Arabia - Accounting',
-    'version': '2.0',
-    'author': 'DVIT.ME',
+    'version': '2.1',
```
Comment (william-andre) [inline addons/l10n_sa/__manifest__.py:5]: You don't need to update the version, do you?

Comment (JulienAlardot) [reply addons/l10n_sa/__manifest__.py:5]: I don't think so, now that I understand more the upgrade process. When would it be considered as a need (for future cases)?

Comment (william-andre) [reply addons/l10n_sa/__manifest__.py]: Almost never.
When you need to write a migration script inside of this module, and not in the `upgrade` repo; so in some very specific cases.

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Comment by william-andre** [comment]
@robodoo r+

**Comment by FlorianGilbert** [comment]
@robodoo r+

**Pull request closed without merging**